src/Model/ModelManagerInterface.php

```diff
@@ -338,4 +340,11 @@ public function getPaginationParameters(DatagridInterface $datagrid, $page);
      * @phpstan-param class-string $class
      */
     public function addIdentifiersToQuery($class, ProxyQueryInterface $query, array $idx);
+
+    // NEXT_MAJOR: Uncomment this method.
+//    /**
+//     * @phpstan-template T
+//     * @phpstan-param class-string<T> $class
+//     */
+//    public function updateFieldDescriptionMapping(string $class, FieldDescriptionInterface $fieldDescription): void;
```

Comment:
What about adding this method to a new FooInterface instead ?
Then ModelManagerInterface could extends FooInterface.

We could move some others methods, like
```
getNewFieldDescriptionInstance
```
which seems also `FieldDescription`-related